jenkins/databricks/test.sh

```diff
@@ -164,6 +165,18 @@ DELTA_LAKE_CONFS="--driver-memory 2g"
 export PYSP_TEST_spark_eventLog_enabled=true
 mkdir -p /tmp/spark-events
 
+rapids_shuffle_smoke_test() {
+    echo "Run rapids_shuffle_smoke_test..."
+
+    # using MULTITHREADED shuffle
+    PYSP_TEST_spark_rapids_shuffle_mode=MULTITHREADED \
+    PYSP_TEST_spark_rapids_shuffle_multiThreaded_writer_threads=2 \
+    PYSP_TEST_spark_rapids_shuffle_multiThreaded_reader_threads=2 \
+    PYSP_TEST_spark_shuffle_manager=com.nvidia.spark.rapids.$SHUFFLE_SPARK_SHIM.RapidsShuffleManager \
+    SPARK_SUBMIT_FLAGS="$SPARK_CONF" TEST_PARALLEL=1 \
+    ./run_pyspark_from_build.sh -m shuffle_test --runtime_env="databricks" --test_type=$TEST_TYPE
```

Comment:
failed CI in DB runtime
```
[2023-02-02T02:09:27.799Z] + ./run_pyspark_from_build.sh -m shuffle_test --runtime_env=databricks --test_type=nightly

[2023-02-02T02:09:27.799Z] /home/ubuntu/test.sh: line 172: ./run_pyspark_from_build.sh: No such file or directory
```

could try `/home/ubuntu/spark-rapids/integration_tests/run_pyspark_from_build.sh` like other command in db script